composer.json

```diff
@@ -24,10 +24,10 @@
     "coverage": "phpunit --coverage-text"
   },
   "require": {
-    "php": "^5.6.0 || ^7.0"
```

Comment:
we can not drop 5.6 version without a new major version - so please readd 5.6. It should work with phpunit 6.x

Reply:
Unfortunately, the PHPUnit ```6.x``` supports PHP version is ```7.0+```.
Please see more details about the [official website](https://phpunit.de/).

Reply:
yes, the only thing necessary is to add php 7.2 to travis.yml. Sorry, I can not accept a breaking change.

Maybe in version 2 I will only support php 7 - also I will be more strict about the types then and I planned a lot of new features.

Reply:
How about creating the new branch named ```2.x``` to support the PHP ```7.0+```?

Reply:
php 7 is supported already

Reply:
I know that and I mean that we drop the ```5.6``` in the future version ```2.x```.

Reply:
yes, we can do this. but i'm unsure if it is necessary to create a branch now. the only change would be the composer.json and the printer (btw. I have an updated version of this printer in other projects like nb-sessions and verja)

Reply:
I really appreciate contributions because I don't have much time to maintain all the projects but your contributions doesn't change much and to discuss then on the PR that this is a breaking change is not saving time for maintaining my projects.